src/services/user.ts

```diff
@@ -57,7 +64,7 @@ export class UserService {
      * @param password
      * @param force Ignore isAuthenticated
      */
-    public async login(username: string, password: string, force = false) {
+    public async login(email: string, password: string, force = false) {
```

Comment:
is login with username not possible anymore? if it's still possible i would prefer an additional method like `loginEmail` or something

Reply:
can't login with username anymore